### PR TITLE
Task 109

### PR DESCRIPTION
**Pull request by SavanDoshi**

### CODE REVIEW

File: tasks/README.md [hunk 1]
```diff
@@ -1,99 +1,100 @@
...
+`task098_conala_list_intersection` | Given a two lists of numbers find the intersection of the two lists | Answer Generation, Arithmetic
+`task_111_asset_sentence_simplification` | Given a sentence, simplify it so it can be understood by non-native English speakers | Generation, Paraphrasing
+`task_112_asset_simple_sentence_identification` | Given two text pieces, choose the one that is simpler and easier to understand by non-native speakers | Answer Generation, Sentence Comparison
+`task102_commongen_sentence_generation` | Given a collection of concepts, use them in a coherent sentence. | Sentence Generation 
+`task103_facts2story_long_text_generation` | Given 5 facts, write a story that incorporates them. | Long Text Generation 
+`task107_splash_question_to_sql` | Generate an SQL statement from a question asking for certain data. | Structured Query Generation
+`task108_ContextualAbuseDetection_classification` | Given a text detect whether it's abusive or not. | Classification
+`task109_SMSspamcollection_SpamSMSdetection` | Classify SMS into spam or ham | Classification
```
Comment (danyaljj) [inline tasks/README.md:100]: I see that you have added this last line. I am troubled that it shows the whole file as changed.

File: tasks/task109_SMSspamcollection_SpamSMSdetection.json [hunk 2]
```diff
@@ -0,0 +1,11545 @@
+{
+    "Contributors": [
+        "Savan Doshi"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Source": [
+        "SMS Spam Collection v.1"
```
Comment (danyaljj) [inline tasks/task109_SMSspamcollection_SpamSMSdetection.json:9]: Add the URL to the dataset.

File: tasks/task109_SMSspamcollection_SpamSMSdetection.json [hunk 3]
```diff
@@ -0,0 +1,11545 @@
...
+    "Contributors": [
+        "Savan Doshi"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Source": [
+        "SMS Spam Collection v.1"
+    ],
+    "Definition": "In this task, you're given Text messages and you need to classify that message into Spam messages or ham (legitimate) messages based on the content of the text message.",
```
Comment (danyaljj) [inline tasks/task109_SMSspamcollection_SpamSMSdetection.json:11]: We need to reformulate this description in a way that: 
 - it defines what input constitutes "spam"? Let's provide a definition. If this is a crowdsourced dataset, we should use the definitions used in their task. 
 - What is "ham"? if it is the opposite of "spam", we can call it "not spam"?
 - The definition should be explicit what we expect as the output.

**Review by danyaljj — CHANGES_REQUESTED** [review]
Left some comments, mainly regarding the reformulation of the definition.

**Comment by swarooprm** [comment]
Thanks for the PR! Looks good. Definition field is too narrow, you need to add more content, think how will you explain spam and ham to another person, this will exactly be your definition.

**Comment by danyaljj** [comment]
@SavanDoshi  pinging the thread to make sure you have seen the comments above.

**Comment by swarooprm** [comment]
@SavanDoshi, a gentle reminder to update your JSON so that we can accept your task.

File: tasks/task109_SMSspamcollection_SpamSMSdetection.json (hunk 3 above)
Comment (SavanDoshi) [reply tasks/task109_SMSspamcollection_SpamSMSdetection.json:11]: I have made the changes in the JASON.

Yes we can call it "not spam", however in the dataset it is called as "ham".

**Comment by swarooprm** [comment]
@SavanDoshi, we will have to close this PR if we don't receive any response.

**Comment by SavanDoshi** [comment]
> @SavanDoshi, we will have to close this PR if we don't receive any response.

I have made the changes regarding updating the definition. But I am not able to fetch the upstream and hence not able to update my file.

**Comment by swarooprm** [comment]
Ok, I understand your concern. Feel free to do gitpull, create a new PR and close this PR if that is easier for you.

**Comment by SavanDoshi** [comment]
> create a new PR and close this PR if that is easier for you.

Okay, I will create a new PR. Because I tried but was not able to resolve the issue.

**Comment by SavanDoshi** [comment]
I am not able to fetch the upstream therefore I closed this pull request. I will create a new pull request for this task.

**Pull request closed without merging**